src/schemas/table_schemas/guide_rna_sequences.json

```diff
@@ -0,0 +1,123 @@
+{
+    "fields": [
+        {
+            "name": "protospacer",
+            "type": "string",
+            "description": "sequence of protospacer",
+            "constraints": {
+                "required": true
+            }
+        },
+        {
+            "name": "targeting",
+            "type": "boolean",
+            "description": "'TRUE' or 'FALSE', depending on whether or not the guide is targeting a locus of the genome relevant to the experiment. If guide is safe-targeting or non-targeting, targeting = False. 'safe-targeting' refers to guides designed to target genomic sites with no annotated function, used as negative controls, if targeting == False, guide_chr, guide_start, guide_end, strand, and pam can be set to NaN",
+            "constraints": {
+                "required": true
+            }
+        },
+        {
+            "name": "type",
+            "type": "string",
+            "description": "flexible tag describing the role of the guide in the experiment",
+            "constraints": {
```

Comment:
i think we should change to:........"constraints": {
                "required": true,
                "enum": [
                    "safe-targeting",
                    "non-targeting",
                    "targeting",
                    "positive control",
                    "negative control",
                    "variant"
                ]
            }